tools/build_variables.bzl

```diff
@@ -457,6 +457,7 @@ libtorch_python_cuda_core_sources = [
     "torch/csrc/cuda/shared/cudart.cpp",
     "torch/csrc/cuda/shared/nvtx.cpp",
     "torch/csrc/cuda/utils.cpp",
+    "torch/csrc/jit/runtime/register_cuda_ops.cpp",
```

Comment:
I had to add the `register_cuda_ops.cpp` in the `libtorch_python_cuda_core_sources` instead of `jit_sources_full` for the `#ifdef USE_CUDA` to work. Let me know if this is not the right place